specs/core/0_beacon-chain.md

```diff
@@ -1252,36 +1252,37 @@ def process_deposit(state: BeaconState,
     Note that this function mutates ``state``.
     """
     # Validate the given `proof_of_possession`
-    assert validate_proof_of_possession(
+    valid_proof = validate_proof_of_possession(
```

Comment:
```suggestion
    proof_is_valid = validate_proof_of_possession(
```

To avoid confusing the reader into thinking the `valid_proof` variable contains a valid proof as opposed to just being a bool.